util/ranger/points.go

```diff
@@ -220,7 +220,7 @@ func (r *builder) buildFromConstant(expr *expression.Constant) []*point {
 	return getFullRange()
 }
 
-func (r *builder) buildFromColumn(expr *expression.Column) []*point {
+func (*builder) buildFromColumn(_ *expression.Column) []*point {
```

Comment:
What? `buildFromColumn` doesn't use column?

Reply:
It is very interesting. I don't know why.

Reply:
remove the useless